Resources/config/converters.xml

```diff
@@ -7,22 +7,22 @@
     <services>
         <service id="Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener" class="Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener" public="false">
             <tag name="kernel.event_subscriber" />
-            <argument type="service" id="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterManager" />
+            <argument type="service" id="sensio_framework_extra.converter.manager" />
             <argument>true</argument>
         </service>
 
-        <service id="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterManager" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterManager" />
+        <service id="sensio_framework_extra.converter.manager" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterManager" />
 
-        <service id="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DoctrineParamConverter" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DoctrineParamConverter" public="false">
+        <service id="framework_extra_bundle.doctrine_param_converter" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DoctrineParamConverter" public="false">
```

Comment:
Lots of services are missing the `sensio_` part in the prefix, so they don't respect the best practices, and are not reverted to the old names